frontend/e2e/page-objects/InquiryPage.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@playwright/test"
+import HelperBase from "./helperBase"
+
+export default class InquiryPage extends HelperBase {
+  async navigateToInquiriesPage() {
+    await this.page.waitForSelector("p.chakra-text.css-0", { timeout: 5000 })
+    await expect(this.page.locator("p.chakra-text.css-0")).toHaveText(
+      "Welcome back, nice to see you again!",
+    )
+    await this.page.getByText("Inquiries").click()
+  }
+
+  async openInquiryForm() {
+    const inquiryTitle = this.page.getByRole("heading", {
+      name: "Inquiries Management",
+    })
+    await expect(inquiryTitle).toHaveText("Inquiries Management")
+    await this.page.getByRole("button", { name: "Inquiry" }).click()
+    await this.page.waitForSelector(".chakra-modal__footer", {
+      state: "visible",
+    })
+    await expect(
+      this.page.getByLabel("Add Inquiry").getByText("Add Inquiry"),
+    ).toHaveText("Add Inquiry")
+  }
+
+  async submitInquiry() {
```

Comment:
a good best practice to follow when naming your tests is:

<functionality>_when<Scenario>_should<ExpectedOutcome>

For example,

addInquiry_whenValidInquiryIsAdded_shouldShowInquiryInInquiriesList()

And so on.

Reply:
(this is from Roy Osherove's Art of Unit Testing)